pom.xml

```diff
@@ -721,6 +721,9 @@
             <ignore>java.nio.file.Path</ignore>
             <ignore>java.io.File</ignore>
           </ignores>
+          <annotations>
+            <annotation>io.netty.util.SuppressJava6Requirement</annotation>
```

Comment:
@jaymode you also need to adjust his one as otherwise the build will fail:

https://garage.netty.io/teamcity/viewLog.html?buildId=27812&buildTypeId=pulls_netty&guest=1 

;)